grassroots-backend/src/vite-env.d.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable check-file/filename-naming-convention */
```

Comment:
Ideally we might share this with the frontend, but I didn't see a nice way of doing so.